scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
     Parameters
     ----------
     cost_matrix : 2d-array, square, non-negative
-        A square adjacency matrix. In this implementation, :math:`A` =
-        `cost-matrix` in the objective function above.
+        The square adjacency matrix :math:`A` in the objective function above.
```

Comment:
Nothing wrong with the way it was, but I try to avoid equating math symbols and code variables.